src/main/scala/de/unruh/isabelle/pure/Transition.scala

```diff
@@ -0,0 +1,109 @@
+package de.unruh.isabelle.pure
+
+import de.unruh.isabelle.control.Isabelle
+import de.unruh.isabelle.mlvalue.Implicits.listConverter
+import de.unruh.isabelle.mlvalue.{MLValue, MLValueWrapper}
+import Transition.Ops
+
+import scala.language.postfixOps
+
+// Implicits
+import de.unruh.isabelle.mlvalue.Implicits._
+import de.unruh.isabelle.pure.Implicits.theoryConverter
+
+/** Represents a transition between proof states (ML type `Transition`) in the Isabelle process.
```

Comment:
The ML type is `Toplevel.transition`, I believe?

Reply:
Yes, fixed.